earth_enterprise/src/common/timeutils_unittest.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 The Open GEE Contributors
```

Comment:
FYI, this is an initial set of unit tests to handle the time subtraction code, which is the trickiest new code. Additional unit tests will be added in a follow up ticket.